README.md

```diff
@@ -161,6 +161,33 @@ Comments:
 * (4) we are importing instances for asynchronous validators into current scope
   so that later we can use `.isValidAsync`/`.validateAsync` extension methods.
 
+#### Another asynchronous monad
+
+At the time there exists possibility to use cats.IO or scalaz.IO monad for asynchronous validation rules.
+First of all proper dependency is required
+```
+libraryDependencies += "com.github.krzemin" %%% "octopus-scalaz-effect" % "0.3.3"
+libraryDependencies += "com.github.krzemin" %%% "octopus-cats-effect" % "0.3.3"
+```
+Then with propper import `asyncF` method on Validator gives user the possibility to specify target monad:
```

Comment:
It should be `asyncM`, if we have `AsyncValidatorM`, don't you think so?

Reply:
Done